### PR TITLE
Fix viewer height and properties padding

### PR DESCRIPTION
**Pull request by LyoCC**
1. fix viewer height
2. fix padding for newlines of the properties

### CODE REVIEW

File: src/Viewer.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const Viewer = ({barColor, formState, language}) => {
 		<>
 			<ThemeProvider theme={typoTheme}>
 			<CssBaseline/>
-			<Grid item xs={12} sm={10} md={5} sx={{border:1, ml:{sm:0,md:4}, mb:{xs:20,sm:20,md:0}, minHeight:{sm:'500px',md:0}, height:(200 + formState.properties.length * 20) + 'px'}} >
+			<Grid container item sx={{border:1}}>
```
Comment (lderequesensS) [inline src/Viewer.jsx:10]: Can we have a smaller box?, when you have only 1 property the white space is kinda big

**Comment by lderequesensS** [comment]
Looking good, I only left a comment but we can fix it later. Feel free to marge it

File: src/Viewer.jsx [hunk 2]
```diff
@@ -18,8 +18,8 @@ export const Viewer = ({barColor, formState, language}) => {
 						<Typography sx={{fontWeight: 'bold'}}>{formState.level}</Typography>
 					</Grid>
 				</Grid>
-				<Grid container direction="column" justifyContent="space-between" alignItems="flex-start" height={'calc(100% - 50px)'}	>
-					<Grid item sx={{width:'98%'}} >
+				<Grid container direction="column" justifyContent="space-between" alignItems="flex-start" height={'calc(100% - 50px)'} sx={{minHeight:'300px'}}	>
```
Comment (lderequesensS) [inline src/Viewer.jsx]: Maybe the question that I did was better written here

Comment (LyoCC) [reply src/Viewer.jsx]: Yes, you can change the minHeight property for the one you want.

**Pull request merged**